### PR TITLE
Fix Julia 1.5 Tests

### PR DESCRIPTION
**Pull request by kanav99**
cc: @utkarsh530 @ChrisRackauckas

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ os:
   - linux
 #  - osx
 julia:
-  - 1.4
+  - 1.5
```
Comment (devmotion) [inline .travis.yml:11]: I guess we might want to change (back?) to
```suggestion
  - 1
```
such that always the stable Julia version is tested automatically.

Comment (kanav99) [reply .travis.yml:11]: We can have a separate testing for this, for this PR, I am aiming to see what issues we have in Julia 1.5 tests. See the travis tests, they are all red

Comment (devmotion) [reply .travis.yml:11]: Travis will use Julia 1.5 when you use `- 1`.

Comment (kanav99) [reply .travis.yml:11]: Ohh Travis follows semver, then we should do it, you are right

**Comment by utkarsh530** [comment]
Probably splitting of AlgConvergence_I is required too? https://travis-ci.org/github/SciML/OrdinaryDiffEq.jl/jobs/716537488

**Comment by kanav99** [comment]
Regression is real, events tests take 35 minutes on travis.

**Comment by ChrisRackauckas** [comment]
Most of the issue was just because coverage now takes longer to compute.

**Pull request merged**